src/shared/ui/section-label/section-label.tsx

```diff
@@ -13,8 +13,8 @@ const sectionLabelVariants = cva(cx('label'));
 
 export const SectionLabel = ({ children, className, ...props }: SectionLabelProps) => {
   return (
-    <h4 {...props} className={sectionLabelVariants({ className })}>
+    <p {...props} className={sectionLabelVariants({ className })}>
       {children}
-    </h4>
+    </p>
```

Comment:
I'd ask for an expert opinion from @Illia-Sakharau before making this change, as I personally disagree that it should be a paragraph.

Reply:
This issue was discussed at today's call, he was not against it. 
But what tag do you suggest? h4 causes accessibility issues. Not all titles have labels, and we are unlikely to be able to build a harmonious hierarchy by leaving them as titles.

Reply:
@SpaNb4, @natanchik
Which element to redo depends only on the semantics that we put into this text.
If it's more of a decorative element, then the `<p>` tag will do fine.
If it is still more of a title without "water" that displays the essence of the section, then I would redo it to a higher-level header, maybe even up to `<h2>`, despite the fact that it is less noticeable than the title.

If I'm not mistaken, we decided that in our case it's more of a decorative element. Therefore, I guess it will not be a mistake to convert it into `<p>` tag.